packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -897,18 +944,23 @@ contract InvestmentPool is
 
         if (milestones[_terminationMilestoneId].paidAmount > 0) {
             tokensReserved = milestones[_terminationMilestoneId].paidAmount;
-        }
-        else {
+        } else {
             tokensReserved = getMilestoneSeedAmount(_terminationMilestoneId);
         }
 
         // TODO: perhaps linearize the logic?
-        return memMilestonePortions[_terminationMilestoneId] - 
-            (tokensReserved * PERCENTAGE_DIVIDER / getTotalMilestoneTokenAllocation(_terminationMilestoneId));
+        // 25% - (60$ * 100% / 300$)
```

Comment:
Maybe we should leave out such calculations from comments, or frame them as an example